From 294290f751afa18ce24d3c059ccb323e9c129b1e Mon Sep 17 00:00:00 2001
From: Dmitry Ornatsky <cdo021c@motorola.com>
Date: Tue, 18 Aug 2015 16:47:36 -0500
Subject: [PATCH 026/959] IKKRNBSP-3333 arm/dt: 8952: fix group size for
 gpio_keys

Array of gpios was reduced but size variable remained
by old size.

Change-Id: Ia2922b80fc196dc3a1e94285b4b6fd862ea1ea86
Signed-off-by: Dmitry Ornatsky <cdo021c@motorola.com>
Reviewed-on: http://gerrit.mot.com/777586
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
index e0f79fed587..c21dbbb33a5 100644
--- a/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-moto-common.dtsi
@@ -23,6 +23,7 @@
 &tlmm_pinmux {
 	tlmm_gpio_key {
 			qcom,pins = <&gp 85>;
+			qcom,num-grp-pins = <1>;
 	};
 };
 
-- 
2.11.0

